src/WebJobs.Script/Description/FunctionMetadata.cs

```diff
@@ -22,6 +22,11 @@ public FunctionMetadata()
         /// </summary>
         public string ScriptFile { get; set; }
 
+        /// <summary>
+        /// Gets or sets the optional named entry point for a function.
+        /// </summary>
+        public string EntryPoint { get; set; }
```

Comment:
We can probably easily support this new metadata property for C# functions as well. The C# `FunctionEntryPointResolver` is already using the "run" name convention that I'm adding for Node. It would be good for the two to behave the same.